### PR TITLE
New wip in azure, new kvm release

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

### CODE REVIEW

File: kvm.yaml [hunk 1]
```diff
@@ -8,14 +8,31 @@
...
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: chart-operator
+      version: 0.7.0
+    - name: cluster-operator
+      provider: kvm
+      version: 0.21.5
```
Comment (jgsqware) [inline kvm.yaml]:
```suggestion
      version: 0.22.0
```

File: kvm.yaml [hunk 2]
```diff
@@ -8,14 +8,31 @@
       version: 0.7.0
     - name: cluster-operator
       provider: kvm
-      version: 0.21.5-dev
+      version: 0.21.6-dev
```
Comment (jgsqware) [inline kvm.yaml]:
```suggestion
      version: 0.22.1-dev
```

File: azure.yaml [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 0.21.6-dev
```
Comment (jgsqware) [inline azure.yaml]:
```suggestion
    version: 0.22.0-dev
```

File: azure.yaml [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+  date: 2019-01-29T13:00:00Z
+  version: 11.0.1
```
Comment (jgsqware) [inline azure.yaml]:
```suggestion
  version: 11.1.0
```

File: azure.yaml [hunk 5]
```diff
@@ -1,3 +1,18 @@
+- active: true
```
Comment (jgsqware) [inline azure.yaml]:
```suggestion
- active: false
```

File: azure.yaml [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 0.22.0-dev
+  date: 2019-01-29T13:00:00Z
+  version: 11.1.0
```
Comment (MarcelMue) [inline azure.yaml]:
```suggestion
  version: 11.0.1
```

File: kvm.yaml [hunk 7]
```diff
@@ -8,14 +8,31 @@
...
+      version: 0.7.0
+    - name: cluster-operator
+      provider: kvm
+      version: 0.22.0
+    - name: flannel-operator
+      version: 0.2.0
+    - name: kvm-operator
+      version: 3.10.0
+  date: 2020-01-29T12:00:00Z
+  version: 11.1.0
```
Comment (MarcelMue) [inline kvm.yaml]:
```suggestion
  version: 11.0.1
```

File: kvm.yaml [hunk 8]
```diff
@@ -8,14 +8,31 @@
...
-      version: 0.21.5-dev
+      version: 0.22.1-dev
     - name: flannel-operator
       version: 0.2.0
     - name: kvm-operator
       version: 3.11.0
-  date: 2020-01-28T12:00:00Z
-  version: 11.1.0
+  date: 2020-01-29T13:00:00Z
+  version: 11.1.1
```
Comment (MarcelMue) [inline kvm.yaml]:
```suggestion
  version: 11.1.0
```

File: azure.yaml (hunk 6 above)
Comment (kopiczko) [reply azure.yaml]: We introduced a new feature of disabling TCP DNS checks. Is this is a bug fix?

Comment (MarcelMue) [reply azure.yaml]: Is it a feature or is it a bugfix? Seemed like a bug affecting a customer to me. Ignore me if wrong.

Comment (jgsqware) [reply azure.yaml]: Not a bug, Customer dosent support TCP DNS. The tool checked per default. we allow to disable this check with this release. So for me it's a feature

**Comment by MarcelMue** [comment]
But some release notes for azure should be written and get approval like here:
https://github.com/giantswarm/releases/tree/master/release-notes/azure

File: release-notes/kvm/v11.1.0.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+## :zap: Giant Swarm Release 11.1.0 for KVM is now active for you! :zap:
+
+###net-exporter v1.6.0
```
Comment (MarcelMue) [inline release-notes/kvm/v11.1.0.md]:
```suggestion
### net-exporter v1.6.0
```

File: release-notes/kvm/v11.1.0.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+## :zap: Giant Swarm Release 11.1.0 for KVM is now active for you! :zap:
+
+### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+
+- Allow to disable TCP DNS check.
```
Comment (tfussell) [inline release-notes/kvm/v11.1.0.md]:
```suggestion
- Allow disabling TCP DNS check.
```

File: azure.yaml [hunk 11]
```diff
@@ -1,3 +1,18 @@
...
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 0.22.0-dev
```
Comment (tfussell) [inline azure.yaml]:
```suggestion
    version: 0.22.1-dev
```

**Comment by tfussell** [comment]
Should this be released on all four platforms? It doesn't seem good to have versions out of sync between providers if we can avoid it.

**Comment by jgsqware** [comment]
> Should this be released on all four platforms? It doesn't seem good to have versions out of sync between providers if we can avoid it.

There is no really need for others. If needed we can do it, it was to remove overhead on release.

**Pull request merged**